config/constants.go

```diff
@@ -99,7 +99,8 @@ const (
 	RoleLabel = "loadtest-role"
 
 	// RunContainerName holds the name of the main container where the test is
-	// executed.
+	// executed. The runtime for the test may contain multiple run containers.
+	// The main container is always the first container on the list.
 	RunContainerName = "run"
```

Comment:
We may want to change this value to "main".

Reply:
Should we also submit `main` in the load test configuration file for name field for each container?